src/pages/Order/components/InfoInput.js

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from 'react';
+
+function InfoInput({ label }) {
+  const [move, setMove] = useState(false);
+
+  const cursorInput = () => {
+    setMove(true);
+  };
+  return (
+    <div className={`orderTextInput${move === true ? ' move' : ''}`}>
```

Comment:
```suggestion
    <div className={`orderTextInput${move ? ' move' : ''}`}>
```
move 라는 state가 이미 boolean 값이기에 위와 같은 방법으로 구현해주셔도 됩니다!